### ISSUE TITLE
Feature request: `focus` imperative handle

### ISSUE THREAD

**Issue by alexstrat** [MuhammedAlmaz/React-Native-INTL-Phone-Input#11]
It would be handy to have a `focus` method on `<IntlPhoneInput />`

```tsx
const inputRef = React.useRef();

// focus on mount
React.useEffect(() => {
  inputRef.current.focus();
}, []);

return (
  <IntlPhoneInput
    ref={inputRef}
    ...
  />
);
```

**Comment by MuhammedAlmaz** [comment]
Its fixed on 1.2.13 version

**Comment by motocombotag** [comment]
@MuhammedAlmaz Not working in 1.2.26.
